client/my-sites/domains/domain-management/list/list-all.jsx

```diff
@@ -436,9 +435,15 @@ class ListAll extends Component {
 
 		const saveWhoisPromises = selectedDomainNamesList.map( ( domainName ) => {
 			const updatedContactInfo = this.getUpdatedContactInfo( domainName, contactInfo );
-			return wpcom
-				.undocumented()
-				.updateWhois( domainName, updatedContactInfo, this.state.transferLockOptOut )
+			return wpcom.req
+				.post( {
+					path: `/domains/${ domainName }/whois`,
+					apiVersion: '1.1',
+					body: {
+						whois: updatedContactInfo,
+						transfer_lock: this.state.transferLockOptOut,
+					},
+				} )
```

Comment:
This is yet another way how to pass `body` to `.post`. It works, but usually we write it as:
```js
wpcom.req.post( `/domains/${ domainName }/whois`, {
  whois,
  transfer_lock,
} );
```
Also, due to a bug in the `wpcom` library, the `apiVersion` param would be ignored because it's expected to be in the `query` object rather than `params`.

Reply:
TIL.  Thank you.

Reply:
Addressed as part of https://github.com/Automattic/wp-calypso/pull/57537/commits/9e37b9f20abe1894fb8b7ddd06c9c172608b95b0.